schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -550,6 +571,130 @@
         "InstallerUrl",
         "InstallerSha256"
       ]
+    },
+    "NestedInstaller": {
+      "type": "object",
+      "properties": {
+        "InstallerLocale": {
+          "$ref": "#/definitions/Locale"
+        },
+        "Platform": {
+          "$ref": "#/definitions/Platform"
+        },
+        "MinimumOSVersion": {
+          "$ref": "#/definitions/MinimumOSVersion"
+        },
+        "Architecture": {
+          "type": "string",
+          "enum": [
+            "x86",
+            "x64",
+            "arm",
+            "arm64",
+            "neutral"
+          ],
+          "description": "The installer target architecture"
+        },
+        "NestedInstallerType": {
+          "type": [ "string", "null" ],
+          "enum": [
+            "msix",
+            "msi",
+            "appx",
+            "exe",
+            "inno",
+            "nullsoft",
+            "wix",
+            "burn",
+            "portable"

```

Comment:
This is actually the correct PR for it, but since I'm thinking of it in the context of archives anyway...

I think we need to consider a `portableWithInit` type (or some other way of achieving it) where the package installer files are intended to stay on the device like portable, but there is also some initialization step to run.  Example: I wrote a very simple .pfx viewer that invoked the existing UI on them, but it needed to be run once in order to add the shell verb to the registry.  This would be a portable installer as the exe was needed after install, but it was more cumbersome if you would have to run it once by hand since afterward you never needed to use the exe directly again.